api/src/pcapi/core/users/email/update.py

```diff
@@ -108,7 +104,6 @@ def request_email_update(user: models.User, new_email: str, password: str) -> No
 
     generate_and_send_beneficiary_confirmation_email_for_email_change(user, new_email)
 
-    check_email_address_does_not_exist(user.email)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Du coup on retire ça complètement ? 
j'aurais plutôt gardé `check_email_address_does_not_exist(new_email)`

Reply:
Il s'agit d'une erreur pendant la réécruture de l historique la verification est faite quelques lignes en dessus